src/frontend/src/test-e2e/flows.ts

```diff
@@ -88,11 +88,9 @@ export const FLOWS = {
     deviceName: string,
     browser: WebdriverIO.Browser
   ): Promise<void> => {
-    const welcomeView = new WelcomeView(browser);
-    await welcomeView.waitForDisplay();
-    await welcomeView.login();
-    await welcomeView.typeUserNumber(userNumber);
-    await browser.$("button[data-action='continue']").click();
+    const authenticateView = new AuthenticateView(browser);
+    await authenticateView.waitForDisplay();
+    await authenticateView.pickAnchor(userNumber);
```

Comment:
Does this e2e depend on logging in at some point before? Or at least on creating the anchor in the same machine without clearing the local storage?

Reply:
Yes, all tests that use the `login` flow star from a different screen now.